draft-ietf-rtcweb-jsep.xml

```diff
@@ -1765,7 +1765,7 @@ Original:
           the rules defined for Trickle ICE. Otherwise, the
           IceCandidate is treated as an end-of-candidates indication,
           as defined in
-          <xref target="RFC8838" format="default"/>.</t>
+          <xref target="RFC8840" sectionFormat="comma" section="8.1"/>.</t>
```

Comment:
I think we want RFC8840, Section 14 here, as the text isn't referring to a=end-of-candidates specifically.

Reply:
RFC8840, Section 14, is the References section. 
Do you mean RFC 8838, Section 14 ("Receiving an End-of-Candidates Indication")?

Reply:
oops, I had meant RFC 8838, as before.

Reply:
Makes sense. I will update the PR.